package.json

```
@@ -12,7 +12,7 @@
     "fmt:check": "prettier --check '**/*.{js,jsx,ts,tsx,json}'",
     "build": "npm run fmt && node ./module/include.js",
     "dev:testnet": "./scripts/dev.sh",
-    "dev:mainnet": "./scripts/dev.sh -a devgovgigs.near -c devgovgigs.near -n mainnet",
+    "dev:mainnet": "./scripts/dev.sh -a devhub.near -c devhub.near -n mainnet",
```

The contract is still under devgovgigs.near account:

```suggestion
    "dev:mainnet": "./scripts/dev.sh -a devhub.near -c devgovgigs.near -n mainnet",
```

---

@frol thank you!